.readthedocs.yaml

```diff
@@ -4,11 +4,10 @@ build:
   os: ubuntu-24.04
   tools:
     python: "3.11"
-  jobs:
-    post_create_environment:
-      - pip install poetry
-    post_install:
-      - VIRTUAL_ENV=$READTHEDOCS_VIRTUALENV_PATH poetry install --with docs
+  commands:
```

Comment:
according to the readthedocs docs, we should lean towards using `build.jobs` stuff instead

https://docs.readthedocs.com/platform/stable/config-file/v2.html#build-commands

> we recommend using [build.jobs](https://docs.readthedocs.com/platform/stable/config-file/v2.html#build-jobs) instead

Reply:
I _think_ I implemented `build.jobs` correctly but it's possible I assigned the commands to the incorrect build stages.

https://docs.readthedocs.com/platform/stable/builds.html

https://docs.readthedocs.com/platform/stable/build-customization.html